commons/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+pub mod env_command;
```

Comment:
```suggestion
#![warn(unused_crate_dependencies)]
#![warn(clippy::pedantic)]

pub mod env_command;
```

Reply:
This review comment isn't resolved

Reply:
This is such a great catch. I must have resolved by accident. This generates a TON of changes needed. Also it's clear work needs to be done in commons. Thanks a ton for the review.

Reply:
Updated, thanks again.